sdk/translation/Azure.AI.Translation.Text/src/Custom/TextTranslationTransliterateOptions.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using System.Collections.Generic;
+
+namespace Azure.AI.Translation.Text
+{
+    /// <summary> Client options for TextTranslationClient.Transliterate </summary>
+    public partial class TextTranslationTransliterateOptions
+    {
+        /// <summary>
+        /// Specifies the language of the text to convert from one script to another.
+        /// Possible languages are listed in the transliteration scope obtained by querying the service
+        /// for its supported languages.
```

Comment:
A user can interpret the suggestion in this comment as "look up the answer online", rather than "call the GetLanguages method and pass 'transliteration' as the `scope` parameter". I would recommend making it clearer. This also applies to other properties below with a similar doc comment, such as `FromScript`, `ToScript`, etc..

Reply:
I have added this to my notes for the API review.